spec/factories/users.rb

```diff
@@ -1,5 +1,10 @@
+# frozen_string_literal: true
+
+require 'faker'
+
 FactoryBot.define do
   factory :user do
-    
+    email { "#{Faker::Lorem.characters(number: 10)}@gmail.com" }
```

Comment:
We can use [`#sequence`](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#sequences) as well to handle unique attributes